ci-operator/step-registry/rosa/cluster/wait-ready/hypershift-infra/rosa-cluster-wait-ready-hypershift-infra-ref.yaml

```diff
@@ -0,0 +1,22 @@
+ref:
+  as: rosa-cluster-wait-ready-hypershift-infra
```

Comment:
@mukrishn After this line, add below section 
  from_image:
    namespace: ci
    name: cli-ocm
    tag: latest